configure.ac

```diff
@@ -1879,9 +1883,12 @@ if test x$bitcoin_enable_qt != xno; then
     echo "    with qr           = $use_qr"
 fi
 echo "  with zmq            = $use_zmq"
-echo "  with test           = $use_tests"
+if test x$enable_fuzz == xno; then
+    echo "  with test     = $use_tests"
+else
+    echo "  with test     = not building test_dash because fuzzing is enabled"
+    echo "    with fuzz   = $enable_fuzz"
 echo "  with bench          = $use_bench"
-if test x$use_tests != xno; then
     echo "  with fuzz           = $enable_fuzz"
 fi
```

Comment:
22069:
```suggestion
if test x$enable_fuzz == xno; then
    echo "  with test     = $use_tests"
else
    echo "  with test     = not building test_dash because fuzzing is enabled"
    echo "    with fuzz   = $enable_fuzz"
fi
echo "  with bench    = $use_bench"
```